core/src/test/java/io/questdb/griffin/LimitTest.java

```diff
@@ -224,6 +224,24 @@ public void testLastN() throws Exception {
         testLimit(expected, expected2, query);
     }
 
+    @Test
```

Comment:
Test content should be wrapped in assertMemoryLeak().

Reply:
good catch! thanks!